### DISCUSSION TITLE
Make a tabcontrol scrollable

### DISCUSSION THREAD

**Discussion by jegan-beckman** [AvaloniaUI/Avalonia#7897] [Q&A]
Hi, I would like to have a tabcontrol that scrolls horizontally using scrollbar buttons('<', '>') within Avalonia.  Would one happen to have any suggestions?  Thank you.

**Comment by maxkatz6** [comment]
To add horizontal buttons to the tabcontrol header, you need to change scroll viewer style used in tab control.
See example https://gist.github.com/maxkatz6/c50cb80c3a1402518a71f2bf5fe07cba. I used it before for my app, but didn't test this piece though.
